sdk-webapp/i18n/en.ts

```diff
@@ -361,14 +361,14 @@
     <name>add_new_target</name>
     <message>
         <source>add new target</source>
-        <translation>Add New Target</translation>
+        <translation>Add new Target</translation>
```

Comment:
It was among the examples mschuele gave :) But to me capitalized New looks correct too.